src/services/api.ts

```diff
@@ -1,42 +1,30 @@
+'use server';
+
 import axios from 'axios';
+import { IUser, CreateReviewType, CreateSellerType } from '@/constants/types';
+import axiosClient, { setAuthToken } from "@/config/client";
 
 const API = axios.create({
   baseURL: process.env.NEXT_PUBLIC_API_URL || 'http://localhost:8001/api/v1'
 });
 
-// Authenticate user
-export const authenticateUser = async (uid, username) => {
-  try {
-    const response = await API.post('/users/authenticate', {
-      auth: {
-        user: {
-          uid,
-          username,
-        },
-      },
-    });
-    return response.data;
-  } catch (error) {
-    console.error('Error authenticating user:', error);
-    throw error;
-  }
-};
 
 // Fetch a single pioneer user
-export const fetchUser = async (userId) => {
+export const fetchUser = async (userId: string) => {
   try {
-    const response = await API.get(`/user/${userId}`);
+    const response = await API.get(`/users/${userId}`);
+    console.log('pioneer user', response.data)
     return response.data;
   } catch (error) {
-    console.error(`Error fetching pioneer user with ID ${userId}:`, error);
-    throw error;
+    // console.error(`Error fetching pioneer user with ID ${userId}:`, error);
+    return {message: 'Error fetching pioneer user with ID',}
```

Comment:
Is there a reason why this error handling was replaced?  Regardless, the return message looks incomplete to me.

Reply:
Okay, I will revert it back.